hazelcast/src/main/java/com/hazelcast/config/NativeMemoryConfig.java

```diff
@@ -214,6 +220,28 @@ public NativeMemoryConfig setMetadataSpacePercentage(float metadataSpacePercenta
         return this;
     }
 
+    /**
+     * Returns the persistent memory (Optane) directory to be used to store memory structures allocated by native
+     * memory manager.
+     * <p>
+     * Default value is {@code null}. It indicates that volatile RAM is being used.
+     * {@code null}
+     */
+    public String getOptaneDirectory() {
```

Comment:
Not sure about naming... 
- Field name is `persistentMemoryDirectory`
- Getter/setter are `[get/set]OptaneDirectory`
- Field javadoc mentions `non-volatile memory`:  `Path to the non-volatile memory directory`

I think we should one, the most generic name without referring any specific implementation of it.

Reply:
I used everywhere `persistentMemoryDirectory` and renamed getter/setter to `OptaneDirectory` to make sure the spring's automatic dependency injection works. In the config we use `optane-directory` which is a requirement in the PRD  (see question #3 and Greg's answer) https://hazelcast.atlassian.net/wiki/spaces/PM/pages/1617036369/Optane+DC+Based+HD+Memory
Personally, I would keep everywhere `persistentMemoryDirectory` and `persistent-memory-directory` in the configuration, but then we have to change the PRD's specification.